assignments/objects.js

```diff
@@ -49,16 +107,52 @@ let example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-let parent = {}
+let parent = {
```

Comment:
Great work with the nested object structure! It looks like you've got it down :+1: